packages/EasyStandard/tests/Rector/PhpDocCommentRector/Fixture/HasCorrectPhpDoc.php.inc

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
The class should be named `AssignmentHasCorrectPhpDoc`. The same goes for other similar classes.

Reply:
Done.

Reply:
The same goes for two other files.

Reply:
Fixed.